docs/how-to/HA.md

```diff
@@ -0,0 +1,681 @@
+    
+4. Create an `init.sql.template` file in newly created directory with the following content:
+    
+    ```sql
+    CREATE DATABASE "pmm-managed";
+    CREATE USER <YOUR_PG_USERNAME> WITH ENCRYPTED PASSWORD '<YOUR_PG_PASSWORD>';
+    GRANT ALL PRIVILEGES ON DATABASE "pmm-managed" TO <YOUR_PG_USERNAME>;
+    CREATE DATABASE grafana;
+    CREATE USER <YOUR_GF_USERNAME> WITH ENCRYPTED PASSWORD '<YOUR_GF_PASSWORD>';
+    GRANT ALL PRIVILEGES ON DATABASE grafana TO <YOUR_GF_USERNAME>;
+    
+    \c pmm-managed
+    
+    CREATE EXTENSION IF NOT EXISTS pg_stat_statements;
+    ```
+    
+    
+5. Use **`sed`** to replace the placeholders with the environment variables and write the output to **`init.sql`**.
+    
+    ```bash
+    sed -e 's/<YOUR_PG_USERNAME>/'"$PG_USERNAME"'/g' \
+        -e 's/<YOUR_PG_PASSWORD>/'"$PG_PASSWORD"'/g' \
+        -e 's/<YOUR_GF_USERNAME>/'"$GF_USERNAME"'/g' \
+        -e 's/<YOUR_GF_PASSWORD>/'"$GF_PASSWORD"'/g' \
+        init.sql.template > init.sql
+    ```
+    
+6. Run the PostgreSQL container.
+
+    You can either run all the services on the same instance or on a seperate instance.
+
+    !!! note alert alert-primary "Note"
+        It is recommended to use absolute paths instead of relative paths for volume mounts.
+
+
+    === "Run services on same instance"
+
+        ```sh
+          docker run -d \
+            --name pg \
+            --network pmm-network \
+            --ip ${PG_HOST_IP} \
+            -p 5432:5432 \
+            -e POSTGRES_PASSWORD=${PG_PASSWORD} \
+            -v /path/to/queries:/docker-entrypoint-initdb.d/ \
+            -v pg_data:/var/lib/postgresql/data \
+            postgres:14 \
+            postgres -c shared_preload_libraries=pg_stat_statements
+        ```
+        
+    === "Run services on a seperate instance"
+    
+        ```sh
+           docker run -d \
+            --name pg \
+            -p 5432:5432 \
+            -e POSTGRES_PASSWORD=${PG_PASSWORD} \
+            -v /path/to/queries:/docker-entrypoint-initdb.d \
+            -v pg_data:/var/lib/postgresql/data \
+            postgres:14
+            postgres -c shared_preload_libraries=pg_stat_statements
+        ```
+    
+    Replace **`/path/to/queries`** with the path to your **`init.sql`** file. This command mounts the **`init.sql`** file to the **`docker-entrypoint-initdb.d`** directory, which is automatically executed upon container startup.
+    
+    
+    !!! note alert alert-primary "Note"
+        - If you run the services on the same instance, the `--network` and `--ip` flags are used to assign a specific IP address to the container within the Docker network created in Step 2. This IP address is referenced in subsequent steps as the PostgreSQL service address.        - In the second case, where the services are running on separate instances, these flags are not necessary as PostgreSQL will bind to the default network interface.
```

Comment:
Please remove:
-  `- In the second case, where the services are running on separate instances, these flags are not necessary as PostgreSQL will bind to the default network interface.`. 
 
In fact, you add already immediately after :
- ` The `--network` and `--ip` flags are not required if the services are running on separate instances, as PostgreSQL will bind to the default network interface.`

Compare this note with the ones you added in previous section. You will notice it is more length due to this repetition. Thanks